refactor: define maxℤ and minℤ manually (#333)
plt-amy/1lab: src/Data/Int/Order/Properties.lagda.md

<!--
```agda
open import 1Lab.Prelude

open import Data.Int.Order
open import Data.Int.Base
open import Data.Nat using (max; max-≤l; max-≤r; max-univ; min; min-≤l; min-≤r; min-univ)
```
-->

```agda
module Data.Int.Order.Properties where
```

<!--
```agda
abstract
```
-->

# Properties of the order of integers

> This module establishes further arithmetic and algebraic properties
> of the integers that involve the order. All proofs are straightforward
> case bashes; there are no further comments.

Reviewer: ```suggestion
This module establishes further order-theoretic properties of the
[[integers]]. All proofs are straightforward case bashes; there are no
further comments.
```
